packages/backend/src/graphql/schema.graphql

```diff
@@ -527,19 +517,9 @@ input CreatePaymentPointerInput {
   publicName: String
 }
 
-type Client implements Model {
-  id: ID!
-  name: String!
-  uri: String!
-  image: String!
-  email: String!
-  keys: [ClientKeys!]!
-  createdAt: String!
-}
-
 type ClientKeys implements Model {
   id: ID!
-  clientId: String!
+  paymentPointerId: String!
```

Comment:
```suggestion
  paymentPointerId: ID!
```
not exactly sure if that is correct, both seem to be used in this file, so probably fine to leave as string